server/test_test.go

```diff
@@ -98,8 +98,15 @@ func require_Error(t *testing.T, err error, expected ...error) {
 	if len(expected) == 0 {
 		return
 	}
+	// Try to strip nats prefix from Go library if present.
+	const natsErrPre = "nats: "
+	eStr := err.Error()
+	if strings.HasPrefix(eStr, natsErrPre) {
```

Comment:
Maybe strip only if the expected errors do not themselves have the `nats: ` prefix (or strip also from the expected errors). Still seeing errors such as:
```
=== RUN   TestJWTJetStreamTiers
    jwt_test.go:5517: require equal, but got: nats: maximum number of streams reached != maximum number of streams reached
--- FAIL: TestJWTJetStreamTiers (0.06s)
```